gluon/globals.py

```diff
@@ -1207,9 +1205,3 @@ def _close(self, response):
                 del response.session_file
             except:
                 pass
-
```

Comment:
ehm.....start from master, else CHECK before committing a file that DROPS a method....